package/src/components/AddressForm/v1/AddressForm.js

```diff
@@ -308,6 +326,21 @@ class AddressForm extends Component {
               <ErrorsBlock names={["phone"]} />
             </Field>
           </ColFull>
+
+          {shouldShowIsCommercialField ? (
```

Comment:
Since you're not outputting anything in the `or` part of the ternary operation you could do the following to make it a little cleaner.

```js
{shouldShowIsCommercialField &&
  <ColFull>
    <Field name="isCommercial" labelFor={isCommercialInputId}>
      <Checkbox
        id={isCommercialInputId}
        name="isCommercial"
        label="This is a commercial address."
        isReadOnly={isSaving}
      />
    </Field>
  </ColFull>
}
```